### ISSUE TITLE
Bug of saving keyframe trajectory to KryFrameTrajectory.txt

### ISSUE THREAD

**Issue by Lmy971109** [udaysankar01/xfeatSLAM#10]
Dear author, when I was testing the TUM dataset, the following error occurred at the end of the program:
"
Starting the Viewer
First KF:0; Map init KF:0
New Map created with 724 points
Shutdown
-------

median tracking time: 0.0613609
mean tracking time: 0.0619002

Saving keyframe trajectory to KeyFrameTrajectory.txt ...
段错误 (核心已转储)

"
 Is this a problem with my computer? Or is it a problem within the program? Do you have this problem?

**Comment by Dmm66-lang** [comment]
Yes, I have the same question. I would like to ask the author, have you encountered this problem before?

**Comment by Dmm66-lang** [comment]
Yes, I have the same question. I would like to ask the author, have you encountered this problem before?

**Comment by udaysankar01** [comment]
First of all, I sincerely apologize for the delayed reply, and thank you reporting the issue. The error seems to be a segmentation fault and I suspect it could be related to saving keyframe trajectory. If that is the case, then it has to be an issue with the original ORB-SLAM3 repo as I mostly worked with image feature extraction and matching parts of the repo, and did not make any changes to the trajectory saving part. Could you please share the full output from the command line so I can better understand the problem?
